src/App.js

```diff
@@ -68,7 +68,10 @@ class App extends Component {
     return (
       <Container className="main-container">
         <Route path="/login" component={LogInContainer} />
-        <Route path="/reset-password" component={PasswordResetContainer} />
```

Comment:
The routes probably have their own reasons but it was a little confusing when the route was forgotPass in the email and reset-password here. Could they be the same?

Reply:
I tried to follow the naming conventions in the other routes. Maybe the link can be changed to the email in backend when its content is finalised?